libs/protos/protos/sp_wsm.proto

```diff
@@ -39,6 +40,8 @@ message WASMRequest {
   // Potentially filled out result from previous step. If this is first step in
   // the pipeline, it will be empty.
   optional bytes input_step = 3;
+
```

Comment:
Add a comment similar to the one above it explaining that IF "dynamic" is set on the current step, the SDK should construct a WASMRequest that sets this optional field (and the Wasm func will use it).

Reply:
Done!